internal/global/global.go

```diff
@@ -0,0 +1,12 @@
+package global
+
+import (
+	"github.com/Checkmarx/kics/internal/metrics"
+)
+
+var (
+	// Metric is the global metrics object
+	Metric = &metrics.Metrics{
```

Comment:
Why not declare this inside metrics package as public member instead creating a package just for this?